### DISCUSSION TITLE
Unit Test - Discussion

### DISCUSSION THREAD

**Discussion by go-christianharrel** [PUP-BSIT/playground-group-vrzn_appdev#13] [Q&A] [answered]
Can anyone explain what is the purpose of Unit Testing?

**Comment by Galope-Jhean** [comment, marked as answer]
it verifies that individual units or components of a software system function correctly in isolation.

**Reply by go-christianharrel** [reply]
TNX. EXEMPTED FOR THE FINALS EXAM!
